/dev/urandom blocks in early boot

real fix is to use rdrand


--- systemd-229/src/basic/random-util.c~	2016-02-11 16:28:00.000000000 +0000
+++ systemd-229/src/basic/random-util.c	2016-06-07 21:23:25.864629615 +0000
@@ -119,10 +119,6 @@
         uint8_t *q;
         int r;
 
-        r = dev_urandom(p, n);
-        if (r >= 0)
-                return;
-
         /* If some idiot made /dev/urandom unavailable to us, he'll
          * get a PRNG instead. */
 
